docs/newbs_getting_started.md

```diff
@@ -120,23 +120,23 @@ NOTE: remember to follow the instructions printed at the end of installation (us
 
 ### ** Windows **
 
-After installing QMK you can set it up with this command:
+After installing QMK, open QMK MSYS and run the following command:
```

Comment:
I'm now second-guessing myself as to how well this scans...

Reply:
It's more specific than before but now reads like a double-positive

Reply:
Could drop the first part?

Reply:
`After installing QMK MSYS, open it and run the following command:` ?

Reply:
Just `Open QMK MSYS and run the following command:`

Reply:
Done.